### ISSUE TITLE
Possible CDO linking issues

### ISSUE THREAD

**Issue by bekozi** [ioos/conda-recipes#700]
I encountered some linking issues with ``cdo`` and Ubuntu. Let me know if I can help out with any more debugging.

``libgrib`` not linking (probably also requires ``build-essential``):
```
$ docker run continuumio/anaconda bash -c "conda install --yes -c ioos cdo python=2.7 && cdo"
cdo: error while loading shared libraries: libgrib_api.so.0: cannot open shared object file: No such file or directory
```

Requires ``build-essential``:
```
$ docker run continuumio/anaconda bash -c "conda install -c ioos cdo python=3 && cdo"
cdo: error while loading shared libraries: libgomp.so.1: cannot open shared object file: No such file or directory
```

This works:
```
$ docker run continuumio/anaconda bash -c "apt-get update --yes && \
                                           apt-get install --yes build-essential && \
                                           conda install -c ioos cdo python=3 && cdo"
```

This line looks a little suspect for ``ecmwf_grib``: https://github.com/SciTools/conda-recipes-scitools/blob/master/ecmwf_grib/meta.yaml#L17. However, I would think that would be problematic for the Python 3 builds not 2.7.

**Comment by ocefpaf** [comment]
I was aware of the `grib_api` issue and I have been looking the other way for a while :unamused:

My solution?  Get rid of it :wink: (See PR #702.)
Most of our users do not use the `grib_api`.  I will re-try that in the future though.

Regarding the `build-essential` dependency: that should go in https://github.com/ioos/conda-recipes/issues/676#issuecomment-167015494.

I will edit the wiki and we will ask @rsignell-usgs to review it later.  (Let's keep this issue open until I do so.)

Thanks for looking into this @bekozi!

**Comment by bekozi** [comment]
> My solution? Get rid of it (See PR #702.)

A fine solution. :smile: 

> Regarding the build-essential dependency: that should go in #676 (comment).

Sounds good!

**Comment by ocefpaf** [comment]
Fixed in #770
